index.js

```diff
@@ -12,4 +13,145 @@ navItems.forEach((navItem) => {
     navMenu.classList.toggle('active');
     hamburger.classList.toggle('active');
   });
+});
+
+const cardsArr = [
+  {
+    name: 'card2',
+    image: 'desktop-img/Project1.jpg',
+    title: 'Professional Art Printing Data',
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required; has been the industry\'s standard',
+    buttons: {
+      langs: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+      other: [['See Live', '', 'Img/see-project-icon.svg'], ['See Source', '', 'Img/source-icon.svg']],
+    },
```

Comment:
- I have added a video below which shows that you have not added a popup for multi-post stories and links for see project and see source.

https://user-images.githubusercontent.com/70269507/162087327-c9be7230-1e91-4554-8003-81ffb6aeea63.mov

- [ ] Kindly add links to the See Live and See Source as its shows the project details you made.
- [ ] Kindly add the popup for the mobile version and desktop version for Multi Post Stories as per the Figma design given below.
- Desktop popup:
<img width="666" alt="Screenshot 2022-04-07 at 1 47 23 AM" src="https://user-images.githubusercontent.com/70269507/162087520-997c69ec-b7ba-4148-920d-5821e9bccf48.png">
- Mobile Popup:
<img width="343" alt="Screenshot 2022-04-07 at 1 45 45 AM" src="https://user-images.githubusercontent.com/70269507/162087570-488cd47e-4aab-43ed-8dab-d0bfd46fcfbc.png">